13_biblioteki/13_2_LAPACK.tex

```diff
@@ -0,0 +1,104 @@
+\section{LAPACK - Linear Algebra Package}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{LAPACK - Linear Algebra Package}
+		\begin{itemize}
+			\item Public domain software
+			\item Fortran77 routines
+			\item 1987 - 1992\dots
+			\item second release 1995
+			\item from HP-486 to CRAY,
+			\item 1 397 384 accesses to 21.01.1998
+		\end{itemize}
+		\textbf{CRPC} - \textbf{C}enter for \textbf{R}esearch on \textbf{P}arallel \textbf{C}omputing: 
+		\begin{itemize}
+			\item Rice,
+			\item Tennessee,
+			\item ORNL.
+		\end{itemize}
+		Supported by:
+		\begin{itemize}
+			\item National Sceince Foundation (NSF),
+			\item Defence Advanced Research Projects Agency (DARPA),
+			\item Departament of Energy (DoE).
+		\end{itemize}
+	\end{frame}
+	\begin{frame}[allowframebreaks]{LAPACK - Linear Algebra Package}
+		\begin{itemize}
+			\item vector,
+			\item shared memory multiprocessors,
+			\item high-performance "superscalar" workstations,
+			\item combines algorithms from LINPACK and EISPACK,
+			\item built on level 1, 2 and 3 BLAS (manufacturers optimized),
+			\item portable FORTRAN 77 (+BLAS): 
+			\begin{itemize}
+				\item 580 K lines, 250 K - library, \\350 K - testing and timing
+			\end{itemize}
+			\item basic problems $\Rightarrow$ blocked algorithms
+			\begin{itemize}
+				\item linear equations systems: $Ax = b$, 
+				\item factorization LU, LQ, QR,
+				\item leas squares: $min_x ||Ax - b ||_2$,
+				\item singular value decomposition: $A = U\sum V^T$,
+				\item eigenvalues and eigenvectors: $Ax = \lambda x$.
+			\end{itemize}
+			\item routines:
```

Comment:
Maybe move 'basic problems' to the same frame as routines? The first frame seems a bit too packed up compared to the second one.